sdcm/provision/aws/capacity_reservation.py

```diff
@@ -56,9 +56,9 @@ def _get_cr_request_based_on_sct_config(params) -> Tuple[dict[str, int], int]:
         return instance_counts, duration
 
     @classmethod
-    def get_cr_from_aws(cls, params) -> None:
+    def get_cr_from_aws(cls, params, force_fetch=False) -> None:
         """Retrieves capacity reservations for given test_id from AWS."""
-        if not cls.is_capacity_reservation_enabled(params):
+        if not cls.is_capacity_reservation_enabled(params) and not force_fetch:
```

Comment:
the logic inside `is_capacity_reservation_enabled` depends on thing that might not be available during the cleanup phase

I think I faced that also when working on dedicated hosts, i.e. not all environment variables are being passed down

i.e. this fix is o.k., but we'll need also align the pipeline to pass the same environment as for the test step
so we won't fall into this pit again